MBBSEmu.Tests/Integration/MBBSEmuIntegrationTestBase.cs

```diff
@@ -13,9 +13,10 @@ namespace MBBSEmu.Tests.Integration
 {
     public class MBBSEmuIntegrationTestBase : IDisposable
     {
-        private readonly string[] _moduleFiles = {"MBBSEMU.DLL", "MBBSEMU.MCV", "MBBSEMU.MDF", "MBBSEMU.MSG"};
+        private static readonly Random RANDOM = new Random();
```

Comment:
Seed `Random()` with `Guid.NewGuid().GetHashCode()` to seed it with a psudo-random number